FND/Accounting/AccountingEquity.rdf

```diff
@@ -68,30 +68,31 @@
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20160201/Accounting/AccountingEquity.rdf version of this ontology was modified per the FIBO 2.0 RFC to rework definitions in support of revised Business Entities, Equities and related ontologies.</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20180801/Accounting/AccountingEquity.rdf version of this ontology was modified to eliminate deprecated elements.</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20190901/Accounting/AccountingEquity.rdf version of this ontology was modified to augment the definition of asset, and add income.</skos:changeNote>
+		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20200101/Accounting/AccountingEquity.rdf version of this ontology was modified to add physical asset and eliminate ambiguity in some definitions.</skos:changeNote>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
 	</owl:Ontology>
 	
 	<owl:Class rdf:about="&fibo-fnd-acc-aeq;Capital">
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-acc-aeq;takesForm"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-acc-cur;hasMonetaryAmount"/>
 				<owl:allValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>capital</rdfs:label>
-		<skos:definition>money and other property of a corporation or other enterprise used in transacting its business</skos:definition>
+		<skos:definition>money and other property of an organization used in transacting its business</skos:definition>
```

Comment:
More complex than this. A certain amount of capital may be required to be held by a bank for risk mitigation and cannot be used for everyday transactions.
Also it's not clear if this definition includes capital assets (machinery etc). See https://www.investopedia.com/terms/c/capital.asp which says "In financial economics, the term MAY be expanded to include a company’s capital assets. "

Reply:
So, again based on consulting an SME and given that this term wasn't used anywhere else in FIBO, it makes more sense to make it a synonym of owner's equity and eliminate the issues.  Note that investopedia is not considered to be a definitive source for this, and Barron's includes different definitions with respect to how the term captial is used depending on perspective - it's a different term with respect to banking generally than in the context of finance.

Reply:
In that case we should make clear what our context for this actually is e.g. via a use case. BTW which volume of Barron's are you referring to "Financial and Investment Terms" or "Banking Terms". And why that one (whichever it is)?

Reply:
Barron's Dictionary of Banking Terms, 2nd definition, where the first is a banking definition and second is finance definition.  The Financial and Investment terms dictionary has 15 terms related to capital, but not for capital on its own.  The issue with respect to how much capital a bank must hold for risk mitigation is called 'capital adequacy', also a separate term from capital on its own.  Again, investopedia is not considered a reliable source for these kinds of definitions according to my sources.